src/Abuse/Adapters/TimeLimit.php

```diff
@@ -202,6 +202,43 @@ protected function hit(string $key, int $time): void
         $this->count++;
     }
 
+    /**
+     * Get all abuse logs
+     *
+     * Returns all the abuse logs that are currently in the DB
+     *
+     * @return array
+     */
```

Comment:
Maybe allow to pass a limit and offset here, and just call getLogs. Might not be a very smart idea to use this method on large DB sets.

Reply:
Done